iceoryx_dds/include/iceoryx_dds/gateway/dds_to_iox.hpp

```diff
@@ -38,13 +37,30 @@ class DDS2IceoryxGateway : public gateway_t
 
   public:
     DDS2IceoryxGateway() noexcept;
+
+    /// @brief Set DDS to iceoryx gateway object
+    /// @param[in] channelFactory the channel factory to propagate data across
     DDS2IceoryxGateway(ChannelFactory channelFactory) noexcept;
+
+    /// @brief Load the provided gateway configuration; a channel is setup if required (no error check on channel setup)
+    /// @param[in] config the gateway configuration to load
     void loadConfiguration(const config::GatewayConfig& config) noexcept;
+
+    /// @brief Discover messages coming from DDS.
+    /// @param[in] msg the discovery message.
     void discover(const capro::CaproMessage& msg) noexcept;
+
+    /// @brief forward data between the two terminals of the channel used by the implementation.
```

Comment:
What are terminals? Where is it defined?

Reply:
As discussed earlier it's now documented in the base class.